### ISSUE TITLE
thor doesn't seem to make more than 105 parallel connections, even with `-C 10000`

### ISSUE THREAD

**Issue by magicgoose** [observing/thor#29]
I'm trying to test my server with the command `thor --masked --amount 10000 -C 10000 ws://localhost:8080/hello/`. The report is the following:

	Online               30299 milliseconds
	Time taken           30299 milliseconds
	Connected            9864
	Disconnected         0
	Failed               136
	Total transferred    13.87MB
	Total received       13.86MB

	Durations (ms):

	                     min     mean     stddev  median max
	Handshaking          5       6387       4898    4768 16523
	Latency              0       3             6       2 112

	Percentile (ms):

	                      50%     66%     75%     80%     90%     95%     98%     98%    100%
	Handshaking          4768    8278    10867   11795   14049   15224   16041   16287   16523
	Latency              2       3       4       5       7       9       12      15      112

	Received errors:

	129x                 connect ECONNRESET 127.0.0.1:8080
	7x                   read ECONNRESET

It says it did (almost) 10k connections. But the server indicates that they were not made all in parallel; instead, at most 105 connections were opened _at the same time._

Is there a way to force `thor` to make a specific number of _parallel_ connections? Because the main reason I need it is to test how many connections my server can keep at the same time.

(Also, FYI — I'm using Jetty for the Websocket server implementation).

**Comment by cuvelierm** [comment]
If anyone knows the answer to this I would also be really interested.

**Comment by rskumar** [comment]
yeah, same happening with me. Around 80 connections opened in parallel. Either its in short batches.

**Comment by PenguinArts** [comment]
New updates at this issue? I am using ratchet, connections reach 400-500 then start to disconnect.
